float64 Finger1Position
float64 Finger2Position
float64 Finger3Position
float64 FingersRotationPosition

float64 Finger1Torque
float64 Finger2Torque
float64 Finger3Torque
float64 FingersRotationTorque

bool Finger1Enable
bool Finger2Enable
bool Finger3Enable
bool FingersRotationEnable
